### PR TITLE
Fixture data with nested components

### LINKED ISSUE #1309: Add fixture data with nested components
**Is your feature request related to a problem? Please describe.**
Before the test suite can be implemented for: https://github.com/bbc/simorgh/issues/1100 to handle nested elements given to us in the data feed, we need example data that captures this scenario.

e.g. an article with a`subheading` block with an `image` block nested in it. 

**Describe the solution you'd like**
Since we currently only run our Cypress tests against our two News & one Persian article in the `/data/prod` directory. We can add this 'nesting' example within one of the existing articles

**Describe alternatives you've considered**
We do not update an existing article, but instead we create a new article that has an example of nested blocks. We would need to ensure that our Cypress tests can run against tests in the `/data/test` directory. 

- Create a new article in `/data/test/news/articles/` with id of the correct format e.g. `c0000000029o.json`
- Article should have:
  - a headline with text and a nested image
  - a subheadline with text and a nested video
  - a text block with a paragraph
  - a subheadline with text and with a nested image
  - a text block with a paragraph

- Add row in the `/data/README.md` file adding this test article and explaining what this test article contains.

**Testing notes**
[Tester to complete]

Dev insight: Visual check by Jim or James that the data structure is correct & the data validator script should pass. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #1309 

**Overall change:** Added fixture data with nested components (Images and Video)

**Code changes:**
- Added Article data with the following
  - headline with text and a nested image
  - subheadline with text and a nested video
  - text block with a paragraph
  - subheadline with text and with a nested image
  - text block with a paragraph
- Updated asynValdateDir test
- Updated Readme

---
Screen Shot 2019-06-20 at 06 40 47

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

**Comment by jimjohnsonrollings** [comment]
I think that instead of :

Headline > Text > Image

We should be doing the Text and Image blocks as siblings, and the text block should continue to contain a paragraph and fragments as normal (and similarly with the subheadlines).

That way we will be able to see that the code has (in the future) moved the Image / Video blocks to be siblings of Headline, rather than just see them in the place of Headline.

**Review by RayNjeri — APPROVED** [review]
LGTM

File: data/test/news/articles/c0000000029o.json [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+    "type": "article",
+    "createdBy": "",
+    "created": 1514808060000,
+    "firstPublished": 1514808060000,
+    "lastPublished": 1514811600000,
+    "lastUpdated": 1514811600000,
+    "passport": {
+      "language": "en-gb",
+      "home": "http://www.bbc.co.uk/ontologies/passport/home/News",
+      "category": "opinion",
```
Comment (twinlensreflex) [inline data/test/news/articles/c0000000029o.json:16]: should this be news? 🤷‍♂ not sure if it matters

**Review by twinlensreflex — APPROVED** [review]
looks good to me! :)

**Comment by staylos92** [comment]
This doesn't need testing by a tester

**Pull request merged**